e2e/cypress/tests/integration/ad_ldap/saml_ldap_sync_id_attrib_spec.ts

```diff
@@ -72,13 +72,13 @@ describe('AD / LDAP', () => {
         const randomId = getRandomId();
         const newFirstName = `Firstname${randomId}`;
         const newLastName = `Lastname${randomId}`;
-        cy.updateLDAPUser({
+        (cy as any).updateLDAPUser({
             ...samlLdapUser,
             firstname: newFirstName,
             lastname: newLastName,
         });
         const admin = getAdminAccount();
-        cy.runLdapSync(admin);
+        cy.runLdapSync(admin as any);
```

Comment:
reason for this error is because the type defined for this function in https://github.com/mattermost/mattermost-webapp/blob/f3ecda80ea47ca2645b4df5b387280a1541cb9d3/e2e/cypress/tests/support/ldap_commands.d.ts#L25 is incorrect. Can you please fix it there to 
```ts
runLdapSync(admin: UserProfile): boolean;
```

Reply:
removed any and fixed func definition